lib/HAL/TargetSystem.cpp

```diff
@@ -80,6 +80,10 @@ llvm::Expected<mlir::ModuleOp>
 TargetInstrument::getModule(mlir::ModuleOp parentModuleOp) {
   for (auto childModuleOp :
        parentModuleOp.getBody()->getOps<mlir::ModuleOp>()) {
+    // Check for a QUIR module. If no quir.nodeType, skip this module.
```

Comment:
A little more detail might be helpful.

```suggestion
    // Check for a QUIR module. If no quir.nodeType, skip this module.
    // This likely indicates it is the `main` module
```